src/facere_sensum/fs.py

```diff
@@ -62,19 +63,35 @@ def command_create(config, log_file):
 # Map of pairs: data connector name / connector module.
 _connectors = {}
 
-def score(metric):
+def _score(metric):
```

Comment:
What leading underscore supposed to mean here?

Reply:
The leading underscore makes the function private for the module, so it is intentional - should have been there up front, actually.